server/state.rs

```diff
@@ -332,6 +352,34 @@ impl ServerState for LocalServerState {
         }
     }
 
+    async fn database_schema_commit(
```

Comment:
In `LocalServerState`, commits are made by simply calling the respective commit method in the database instance.